docs/datadog_monitor.md

```diff
@@ -0,0 +1,68 @@
+# Getting Started
+
+This procedure describes the simplest and fastest way to deploy a `DatadogMonitor` with the Datadog Operator.
+
+## Prerequisites
+
+Using the Datadog Operator requires the following prerequisites:
+
+- **Kubernetes Cluster version >= v1.14.X**: Tests were done on versions >= `1.14.0`. However, it should work on versions `>= v1.11.0`. For earlier versions, due to limited CRD support, the Operator may not work as expected.
+- [`Helm`][1] for deploying the `datadog-operator`.
+- [`Kubectl` cli][2] for installing a `DatadogMonitor`.
+
+## Using DatadogMonitor
+
+To deploy a `DatadogMonitor` with the Datadog Operator, see the [`datadog-operator`][3] Helm chart.
+Here are the steps:
+
+1. Update the [values.yaml][4] file of the Datadog Operator Helm chart to include your [Datadog API and application keys][5]
+
+1. Install the [Datadog Operator][6]:
+
+   ```shell
+   helm repo add datadog https://helm.datadoghq.com
+   helm install my-datadog-operator datadog/datadog-operator -f values.yaml
+   ```
+
```

Comment:
we could provide an alternative `helm install` command here as a single step (by passing the creds via the `--set` option)

Reply:
note that the datadog monitor controller is [disabled by default](https://github.com/DataDog/datadog-operator/blob/master/main.go#L91) (at least in the next operator release). So we should make the necessary changes in helm chart and update this step accordingly